tasks/reference.js

```diff
@@ -18,22 +18,29 @@ var revReplace = require('gulp-rev-replace');
 
 gulp.task('reference', ['clean-dist', 'less'], function() {
 
-    var versions = ['1.7', '2.0'];
-    // We construct a reference index file and a complete reference file for each PIM version: 1.7 and 2.0.
-    // When we construct the 1.7 files, we filter to not include the 2.0 only endpoints.
+    var versions = ['1.7', '2.0', '2.1'];
+    // We construct a reference index file and a complete reference file for each PIM version: 1.7, 2.0 and 2.1.
+    // When we construct the 1.7 files, we filter to not include the new 2.0 and the 2.1 endpoints.
+    // Same thing when we construct the 2.0 files, we filter to not include the 2.1 endpoints.
     _.forEach(versions, function(version) {
-        var htmlReferenceIndexfileName = (version === '1.7') ? 'api-reference-index-17' : 'api-reference-index';
+
+        var htmlReferenceIndexfileName = (version === '1.7') ? 'api-reference-index-17' : 
```

Comment:
so basically, we will have to modify it at each minor release with new endpoints in the API?

Reply:
Yep, not really convenient, I know. But it will have to wait to make something better.